payment/src/main/java/com/kyriba/conference/payment/domain/dto/TicketDto.java

```diff
@@ -0,0 +1,36 @@
+package com.kyriba.conference.payment.domain.dto;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.kyriba.conference.payment.domain.DiscountType;
+import com.kyriba.conference.payment.domain.PaymentMethodType;
+import com.kyriba.conference.payment.domain.vo.Amount;
+import lombok.*;
+
+import java.time.LocalDateTime;
+
+/**
+ * @author Igor Lizura
+ */
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+public class TicketDto {
```

Comment:
The same set of comments as for another DTO.

Reply:
done